client/src/pages/Restaurant/FoodList/FoodCard.js

```diff
@@ -1,5 +1,56 @@
-function FoodCard() {
-  return <div>FoodCard</div>;
+function FoodCard({ name, price, description, imageUrl }) {
+  const classes = {
+    cardContainer: {
+      boxSizing: "border-box",
+      width: "100%",
+      border: "2px solid #5a5a5a",
+      padding: "0.125rem",
+      display: "flex",
+      gap: "0.25rem",
+      backgroundColor: "hsl(35,0%,97%)",
+    },
+    foodName: {
+      overflow: "hidden",
+      whitespace: "nowrap",
+      textOverflow: "ellipsis",
```

Comment:
Looks like this cuts off the food name if it gets too long.. Is this intended? If it doesn't cut it off, then prob not needed